config/PropertyNames.xml

```diff
@@ -342,13 +148,18 @@ generated from the section label.
     </section>
 
     <section name="IceMX">
-        <property name="Metrics.[any]" languages="cpp,csharp,java" class="mx" />
+        <property name="Metrics.[any].GroupBy" languages="cpp,csharp,java" />
+        <property name="Metrics.[any].Map" languages="cpp,csharp,java" />
+        <property name="Metrics.[any].RetainDetached" default="10" languages="cpp,csharp,java" />
+        <property name="Metrics.[any].Accept" languages="cpp,csharp,java" />
+        <property name="Metrics.[any].Reject" languages="cpp,csharp,java" />
+        <property name="Metrics.[any]" languages="cpp,csharp,java" />
```

Comment:
You can set Metrics.FooBar=1 and it means something?

Reply:
You can set it yes, and sure that it means anything. From what I can tell the MX properties are a mess. 

We set things like `"IceMX.Metrics.View." + map + "Accept.endpointPort"`. 

From what I can tell we likely dont' use any of this property code for MX right now.

https://github.com/zeroc-ice/ice/blob/main/cpp/src/Ice/MetricsAdminI.cpp/#L26-L33